pkg/node/compute.go

```diff
@@ -122,50 +122,9 @@ func NewComputeNode(
 		},
 	})
 
-	semanticBidStrat := bidstrategy.WithSemantics(config.BidSemanticStrategy)
-	if config.BidSemanticStrategy == nil {
-		semanticBidStrat = bidstrategy.WithSemantics(
-			semantic.NewNetworkingStrategy(config.JobSelectionPolicy.AcceptNetworkedJobs),
-			semantic.NewTimeoutStrategy(semantic.TimeoutStrategyParams{
-				MaxJobExecutionTimeout:                config.MaxJobExecutionTimeout,
-				MinJobExecutionTimeout:                config.MinJobExecutionTimeout,
-				JobExecutionTimeoutClientIDBypassList: config.JobExecutionTimeoutClientIDBypassList,
-			}),
-			semantic.NewStatelessJobStrategy(semantic.StatelessJobStrategyParams{
-				RejectStatelessJobs: config.JobSelectionPolicy.RejectStatelessJobs,
-			}),
-			semantic.NewProviderInstalledStrategy(
-				publishers,
-				func(j *models.Job) string { return j.Task().Publisher.Type },
-			),
-			semantic.NewStorageInstalledBidStrategy(storages),
-			semantic.NewInputLocalityStrategy(semantic.InputLocalityStrategyParams{
-				Locality: config.JobSelectionPolicy.Locality,
-				Storages: storages,
-			}),
-			semantic.NewExternalCommandStrategy(semantic.ExternalCommandStrategyParams{
-				Command: config.JobSelectionPolicy.ProbeExec,
-			}),
-			semantic.NewExternalHTTPStrategy(semantic.ExternalHTTPStrategyParams{
-				URL: config.JobSelectionPolicy.ProbeHTTP,
-			}),
-			executor_util.NewExecutorSpecificBidStrategy(executors),
-		)
-	}
-
-	resourceBidStrat := bidstrategy.WithResources(config.BidResourceStrategy)
-	if config.BidResourceStrategy == nil {
-		resourceBidStrat = bidstrategy.WithResources(
-			resource.NewMaxCapacityStrategy(resource.MaxCapacityStrategyParams{
-				MaxJobRequirements: config.JobResourceLimits,
-			}),
-			resource.NewAvailableCapacityStrategy(ctx, resource.AvailableCapacityStrategyParams{
-				RunningCapacityTracker:  runningCapacityTracker,
-				EnqueuedCapacityTracker: enqueuedCapacityTracker,
-			}),
-			executor_util.NewExecutorSpecificBidStrategy(executors),
-		)
-	}
+	// TODO(forrest) fix before merging
+	// semanticBidStrat := bidstrategy.WithSemantics(config.BidSemanticStrategy)
+	// resourceBidStrat := bidstrategy.WithResources(config.BidResourceStrategy)
```

Comment:
TODO(forrest) fix before merging

Reply:
whoops, Ive fixed the TODO, just forgot to remove it, thanks.